tutorials/tutorial_views.py

```diff
@@ -0,0 +1,944 @@
+
+    # Maximizing the window
+    try:
+        if platform == 'linux' or platform == 'linux2':
+            pag.hotkey('winleft', 'up')
+        elif platform == 'darwin':
+            pag.hotkey('ctrl', 'command', 'f')
+        elif platform == 'win32':
+            pag.hotkey('win', 'up')
+    except Exception:
+        print("\nException : Enable Shortcuts for your system or try again!")
+    pag.sleep(2)
+    pag.hotkey('ctrl', 'h')
+    pag.sleep(2)
+
+    # Shfting topview window to upper right corner
+    try:
+        x, y = pag.locateCenterOnScreen(f'{dir_path}add_waypoint.png')
+        pag.click(x, y - 56, interval=2)
+        if platform == 'win32' or platform == 'darwin':
+            pag.dragRel(525, -110, duration=2)
+        elif platform == 'linux' or platform == 'linux2':
+            pag.dragRel(910, -25, duration=2)
+        pag.move(None, 56)
+        add_tv_x, add_tv_y = pag.position()
+        pag.move(-486, -56, duration=1)
+        pag.click(interval=1)
+        if platform == 'win32' or platform == 'linux' or platform == 'linux2':
+            pag.hotkey('ctrl', 'v')
+        elif platform == 'darwin':
+            pag.hotkey('command', 'v')
+        pag.sleep(4)
+        # Shifting Sideview window to upper left corner.
+        try:
+            x1, y1 = pag.locateCenterOnScreen(f'{dir_path}add_waypoint.png')
+            if platform == 'win32' or platform == 'darwin':
+                pag.moveTo(x1, y1 - 56, duration=1)
+                pag.dragRel(-494, -177, duration=2)
+            elif platform == 'linux' or platform == 'linux2':
+                pag.moveTo(x1, y1 - 56, duration=1)
+                pag.dragRel(-50, -30, duration=2)
+            pag.sleep(2)
+            if platform == 'linux' or platform == 'linux2':
+                pag.keyDown('altleft')
+                pag.press('tab')
+                pag.press('right')
+                pag.keyUp('altleft')
+            elif platform == 'win32':
+                pag.keyDown('alt')
+                pag.press('tab')
+                pag.press('right')
+                pag.keyUp('alt')
+            elif platform == 'darwin':
+                pag.press('command', 'tab', 'right')
+            pag.sleep(1)
+        except (ImageNotFoundException, OSError, Exception):
+            print("Exception: \'Side View Window Header\' was not found on the screen")
+    except (ImageNotFoundException, OSError, Exception):
+        print("Exception: \'Topview Window Header\' was not found on the screen")
+
+    # Adding waypoints
+    if add_tv_x is not None and add_tv_y is not None:
+        pag.sleep(1)
+        pag.click(add_tv_x, add_tv_y, interval=2)
+        pag.move(-50, 150, duration=1)
+        pag.click(interval=2)
+        pag.sleep(1)
+        pag.move(65, 65, duration=1)
+        pag.click(interval=2)
+        pag.sleep(1)
+
+        pag.move(-150, 30, duration=1)
+        x1, y1 = pag.position()
+        # print(x1, y1)
```

Comment:
remove not needed comments

Reply:
Again by mistake. I will remove those. Sorry!